bolt-project.yaml

```
@@ -1,2 +1,12 @@
 ---
 name: peadm
+
+modules:
+- name: puppet/format
+
+- name: puppetlabs/apply_helpers
+- name: puppetlabs/bolt_shim
+- name: puppetlabs/service
+- name: puppetlabs/stdlib
+
+- name: WhatsARanjit/node_manager
```

We need a slightly newer version of node_manager, still waiting on a release to be cut

```suggestion
- name: 'WhatsARanjit/node_manager'
  git: 'https://github.com/WhatsARanjit/puppet-node_manager.git'
  ref: '3775a9d07bdea9a9258225dc7053118b1f2fd871' # 0.7.4-1-g3775a9d
```